pkg/get/get_test.go

```diff
@@ -914,6 +914,53 @@ func Test_DownloadDigitalOcean(t *testing.T) {
 	}
 }
 
+func Test_DownloadEKSCTL(t *testing.T) {
+	tools := MakeTools()
+	name := "eksctl"
+
+	tool := getTool(name, tools)
+
+	const toolVersion = "v0.79.0"
+	const urlTemplate = "https://github.com/weaveworks/eksctl/releases/download/v0.79.0/eksctl_%s_%s.%s"
+
+	tests := []test{
+		{os: "mingw64_nt-10.0-18362",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     fmt.Sprintf(urlTemplate, "Windows", "amd64", "zip")},
```

Comment:
I know you think this is better, but these URLs are subject to change and don't always follow a pattern. Since we cannot control the project releases, please repeat each URL as per the example I sent you for KinD.

Using the full URL means we can copy/paste directly from a GitHub repo URL to compare. This indirection may been clever and more advanced, but in experience it works against us for maintenance.

Reply:
No problem. I will set the url directly.